sdk/communication/Azure.ResourceManager.Communication/src/Azure.ResourceManager.Communication.csproj

```diff
@@ -5,11 +5,9 @@
     <Description>Azure management client SDK for Azure resource provider Microsoft.Communication</Description>
     <PackageTags>azure;management;communication</PackageTags>
     <Nullable>enable</Nullable>
-
     <!--
-    AZC0001 - Communication namespace needs to be registered in the analyzer
-    AZC0008 - ClientOptions complains about a missing nested enum
+    AZC0001 - Azure.ResourceManager.Communication namespace needs to be registered in the analyzer
     -->
-    <NoWarn>$(NoWarn);AZC0001;AZC0008</NoWarn>
+    <NoWarn>$(NoWarn);AZC0001</NoWarn>
```

Comment:
What about this one?

Reply:
Ah, it's the ResourceManager my bad.

Reply:
It seems that the track 2 ARM namespaces are being handled differently and we should add this to the tool as well. I wonder if it makes sense to change this: https://github.com/RezaJooyandeh/azure-sdk-tools/blob/e5a5035fe3cbc9fd4c96cd2766d82155d317260e/src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ClientAssemblyNamespaceAnalyzer.cs#L13 to not have the `Azure` prefix and then allow both namespaces `Azure.X` and `Azure.ResourceManager.X`. Thoughts?